modules/apps/notification/notification-service/src/main/java/com/liferay/notification/internal/term/contributor/NotificationTermProviderRegistryImpl.java

```diff
@@ -0,0 +1,48 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.notification.internal.term.contributor;
```

Comment:
@gabrielwas should we change this package to be `term.provider`?

Reply:
Thx!